openfecwebapp/views.py

```diff
@@ -67,6 +68,17 @@ def render_legal_mur(mur):
         parent='legal'
     )
 
+def render_legal_ao_landing():
+    today = datetime.date.today()
+    ao_min_date = today - datetime.timedelta(weeks=26)
+    results = api_caller.load_legal_search_results(query='', query_type='advisory_opinions', ao_min_date=ao_min_date)
+    recent_aos=OrderedDict(sorted(results['advisory_opinions'].items(), key=lambda item: item, reverse=True))
```

Comment:
You're welcome!  It does sound like what you want based on what you've described - if the recent AOs should be in reverse chronological order then it looks like you're already good to go with things as they are. :-)